testnets/archwaytestnet/assetlist.json

```diff
@@ -53,6 +53,152 @@
         "website": "https://archway.io/",
         "twitter": "https://twitter.com/archwayHQ"
       }
+    },
+    {
+      "description": "Wrapped CONST (Archway Testnet token)",
+      "type_asset": "cw20",
+      "address": "archway1dz6h2smr432gvzruklk3lps7ak2ca7cngdge4vym93wehkkypwpqgwr2q2",
+      "denom_units": [
+        {
+          "denom": "cw20:archway1dz6h2smr432gvzruklk3lps7ak2ca7cngdge4vym93wehkkypwpqgwr2q2",
+          "exponent": 0
+        },
+        {
+          "denom": "wARCH",
+          "exponent": 18
+        }
+      ],
+      "base": "cw20:archway1dz6h2smr432gvzruklk3lps7ak2ca7cngdge4vym93wehkkypwpqgwr2q2",
+      "name": "Wrapped ARCH",
+      "display": "wCONST",
```

Comment:
wCONST is not defined as a denom_unit. You've defined wARCH and could use that...